lib/ibm/cloud/sdk/version.rb

```diff
@@ -1,7 +1,7 @@
 module IBM
   module Cloud
     module SDK
-      VERSION = "0.1.6"
+      VERSION = "0.1.7"
```

Comment:
FYI: I think the usual practice is to push version updates directly instead of through a PR.

Reply:
Correct,  this lets us group whatever PRs we decide should go into a version together